src/client/containers/PullRequestSummary/BuildSection/index.js

```diff
@@ -39,15 +39,16 @@ class BuildSection extends PureComponent {
     if (this.props.repository !== nextprops.repository ||
       this.props.revision !== nextprops.revision) {
       this.props.dispatch(
-        fetchLatestBuildsForRevision(nextprops.repository, nextprops.revision))
+
+        fetchLatestBuildsForRevision(nextprops.repository, '29a7efaab3ad6a82214c203f6c21553eb1571f5f'))
     }
```

Comment:
Apart from these it looks very nice; I really like the added overview it brings.